collect_app/src/main/res/values/strings.xml

```diff
@@ -348,16 +348,37 @@
     <string name="gps_enable_message">GPS is disabled in your device. Would you like to enable it?</string>
     <string name="select_offline_layer">Select Offline Layer</string>
     <string name="polygon_or_polyline">"Save GeoTrace As"</string>
-    <string name="minutes">Minutes</string>
     <string name="geo_clear_warning">Feature already created. Would you like to clear the feature?</string>
     <string name="geo_exit_warning">Discard changes and return to ODK?</string>
     <string name="polygon_validator">Must have at least 3 points to create Polygon</string>
     <string name="polyline_validator">Must have at least 2 points to create Polyline</string>
     <string name="clear">Clear</string>
-    <string name="manual_mode">Manual Mode</string>
-    <string name="automatic_mode">Automatic Mode</string>
+    <string name="manual_mode">Manual recording</string>
+    <string name="automatic_mode">Automatic recording</string>
+    <string name="recording_interval">Recording interval:</string>
+    <plurals name="number_of_seconds">
+        <item quantity="one">%d second</item>
+        <item quantity="other">%d seconds</item>
+    </plurals>
+    <plurals name="number_of_minutes">
+        <item quantity="one">%d minute</item>
+        <item quantity="other">%d minutes</item>
+    </plurals>
+    <string name="accuracy_requirement">Accuracy requirement:</string>
+    <string name="none">None</string>
+    <plurals name="number_of_meters">
+        <item quantity="one">%d meter</item>
+        <item quantity="other">%d meters</item>
+    </plurals>
     <string name="geoshape_instruction">Long press to place marks.</string>
-    <string name="geotrace_instruction">Wait for lock, then tap add marker button to start.</string>
+    <string name="geotrace_location_status_searching">Searching for GPS location, please wait&#8230;</string>
+    <string name="geotrace_location_status_accuracy">GPS location accuracy: %.1f m</string>
+    <string name="geotrace_location_status_acceptable">GPS location accuracy: %.1f m (acceptable)</string>
+    <string name="geotrace_location_status_unacceptable">GPS location accuracy: %.1f m (unacceptable)</string>
+    <string name="geotrace_collection_status_paused">Points recorded: %d</string>
+    <string name="geotrace_collection_status_manual">Points recorded: %d (manual recording)</string>
+    <string name="geotrace_collection_status_auto">Points recorded: %1$d (auto recording, %2$d s)</string>
+    <string name="geotrace_collection_status_auto_accuracy">Points recorded: %1$d (auto recording, %2$d s, %3$d m)</string>
```

Comment:
I suggest making these strings more explicit, e.g. `auto recording every X s, Y m accuracy`. Have you tested what happens if they wrap to two lines in other languages?

Reply:
Good catch!  I fixed the layout to allow for the status lines to have variable height.

I'd like to try to get the English text to fit on one line, so I shortened your suggestion to `recording every X s`; it seems like `every X s` should be enough to imply that it happens automatically.